src/Microsoft.Docs.Template/Views/Template/BuildItemListPartial.cshtml

```diff
@@ -3,8 +3,23 @@
 
 @using Microsoft.Docs.Build;
 @using Newtonsoft.Json.Linq;
+@using System.Text.RegularExpressions;
 @model LandingDataItem
 
+@functions
+{
+    private string removeTags(string html) {
+        string oldHtml;
+        while (true) {
+            oldHtml = html;
+            html = Regex.Replace(html, @"<(?:!--(?:(?:-*[^->])*--+|-?)|script\b(?:[^""'>]|""[^""]*""|'[^']*')*>[\s\S] *?<\/ script\s *| style\b(?:[^""'>]|""[^""]*""|'[^']*')*>[\s\S]*?<\/style\s*|\/?[a-z](?:[^""'>]|""[^""]*""|'[^']*')*)>", " ", RegexOptions.IgnoreCase);
```

Comment:
do we have max line length limitation?

Reply:
it's scaring me....

Reply:
best practice for RegEx is always to specify a timeout...